pom.xml

```diff
@@ -128,6 +130,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
+            <version>2.20.0-420.v8a_08b_d57ca_05</version>
```

Comment:
If you bump the BOM version to `5543.vfd758c7c868d` you won't need the version here.